Source/ACE.Server/Factories/Enum/WealthRating.cs

```diff
@@ -0,0 +1,15 @@
+namespace ACE.Server.Factories.Enum
+{
+    public enum WealthRating
+    {
+        Undef,
+        Shoddy,
+        Poor,
+        Medium,
+        Good,
+        Rich,
+        Incomparable,
+        Exceptional,
+        Phenomenal
+    }
```

Comment:
Maybe Comment what tiers are for each of those for others?

Reply:
I think WealthRating might be used for different things

For this initial function that was added, yes it directly maps tiers to wealth ratings. ie. 8 wealth ratings, 8 tiers